### ISSUE TITLE
Agenda build error on 1.1.2 on ST4

### ISSUE THREAD

**Issue by smithx** [ihdavids/orgextended#12]
- OrgExtended 1.1.2
- ST 4 : 4098

```
Traceback (most recent call last):
  File "C:\Program Files\Sublime Text\Lib\python33\sublime_plugin.py", line 515, in create_text_commands
    o = cls(view)
TypeError: __init__() missing 1 required positional argument: 'prefix'

The above exception was the direct cause of the following exception:

RuntimeError: unable to instantiate 'OrgExtended.orgediting.OrgBaseTimestampCommand'
Traceback (most recent call last):
  File "C:\Program Files\Sublime Text\Lib\python33\sublime_plugin.py", line 1494, in run_
    return self.run(edit)
  File "C:\Users\user1\AppData\Roaming\Sublime Text\Installed Packages\OrgExtended.sublime-package\orgagenda.py", line 1250, in run
  File "C:\Users\user1\AppData\Roaming\Sublime Text\Installed Packages\OrgExtended.sublime-package\orgagenda.py", line 1195, in __init__
  File "C:\Users\user1\AppData\Roaming\Sublime Text\Installed Packages\OrgExtended.sublime-package\orgagenda.py", line 380, in __init__
  File "C:\Users\user1\AppData\Roaming\Sublime Text\Installed Packages\OrgExtended.sublime-package\orgagenda.py", line 485, in SetupView
  File "C:\Users\user1\AppData\Roaming\Sublime Text\Installed Packages\OrgExtended.sublime-package\orgagenda.py", line 1223, in FilterEntries
  File "C:\Users\user1\AppData\Roaming\Sublime Text\Installed Packages\OrgExtended.sublime-package\orgagenda.py", line 565, in FilterEntries
  File "C:\Users\user1\AppData\Roaming\Sublime Text\Installed Packages\OrgExtended.sublime-package\orgagenda.py", line 1019, in FilterEntry
  File "C:\Users\user1\AppData\Roaming\Sublime Text\Installed Packages\OrgExtended.sublime-package\orgagenda.py", line 199, in IsToday
AttributeError: 'OrgDateScheduled' object has no attribute 'after'
```

**Comment by ihdavids** [comment]
Huh.
Thanks 
I will investigate tomorrow.

**Comment by ihdavids** [comment]
This should be fixed now in 1.1.3
